pkg/asset/machines/worker.go

```diff
@@ -24,16 +24,29 @@ import (
 	openstacktypes "github.com/openshift/installer/pkg/types/openstack"
 	vspheretypes "github.com/openshift/installer/pkg/types/vsphere"
 	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	"github.com/pkg/errors"
```

Comment:
nit: can we keep the import convention as-is
```
<internal>

<external packages>

<installer packages>
```

and sorry for wrong placement for `mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"` from https://github.com/openshift/installer/commit/e4d5ec92001c7efe8020b8d3a43188591d7bf9b5

Reply:
> nit: can we keep the import convention as-is

Done with 75f00f130 -> 6723bbd15, although personally I don't see a need to distinguish between other installer packages and other external packages.